tests/js/common/lib/check-potentia-grades.js

```diff
@@ -0,0 +1,35 @@
+/* globals require */
+/* eslint-env browser */
+var fluid  = fluid  || {};
+
+(function (fluid) {
+    "use strict";
+    if (!fluid.identity) {
+        fluid = require("infusion");
+    }
+
+    var gpii = fluid.registerNamespace("gpii");
+
+    require("../../../../");
```

Comment:
Generally nicer to write fluid.require("%gpii-json-schema") if you can

Reply:
Thanks for catching that, it's fixed.